From d395d7616d089eae34c5b2a81e5cefa13c32acb1 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Micha=C5=82=20G=C3=B3rny?= <mgorny@gentoo.org>
Date: Fri, 13 Jul 2012 16:15:14 +0200
Subject: [PATCH] udev: add /lib/udev/rules.d to rules directories

This adds /lib if split-usr is enabled
to the directories where udev searches for rules.d.

This is needed if split-usr is enabled because some software still
installs rules in /lib/udev/rules.d.
---
 src/udev/udev-rules.c |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/src/udev/udev-rules.c b/src/udev/udev-rules.c
index d3b33e4..f30fa5a 100644
--- a/src/udev/udev-rules.c
+++ b/src/udev/udev-rules.c
@@ -1760,6 +1760,9 @@ struct udev_rules *udev_rules_new(struct udev *udev, int resolve_names)
         rules->dirs = strv_new(SYSCONFDIR "/udev/rules.d",
                                "/run/udev/rules.d",
                                UDEVLIBEXECDIR "/rules.d",
+#ifdef HAVE_SPLIT_USR
+                               "/lib/udev/rules.d",
+#endif
                                NULL);
         if (!rules->dirs) {
                 log_error("failed to build config directory array");
-- 
1.7.8.6

